package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hubot-foursquare-lunch",
   "description": "Use Foursquare API to pick a lunch spot",
-  "version": "1.0.1",
+  "version": "1.1.2",
```

Comment:
This is usually updated post merge using the Grunt script. That script will publish the changes as a tag to GitHub, and push it up to NPM in a single operation.